scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -175,12 +194,28 @@ <h3 class="subnav__page-title sd-flex-no-grow sd-empty">
                                 <li ng-if="type === 'monitoring'">
                                     <button ng-click="aggregate.edit('reorder', true)">
                                         <i class="icon-switches"></i>
-                                        &nbsp;<span>{{:: 'Rearrange Groups' | translate}}</span>
+                                        &nbsp;<span>{{ gettext('Rearrange Groups') }}</span>
                                     </button>
                                 </li>
                             </ul>
                         </div>
-            
+                    </div>
+                    <!-- Active filter tags -->
+                    <div class="search-tags" ng-if="aggregate.activeFilters.contentProfile.length > 0">
+                        <ul class="search-tags__list">
+                            <li ng-repeat="(type, keys) in aggregate.activeFilterTags">
+                                <span>{{ type | translate }}:</span>
```

Comment:
not sure about this `| translate` here

Reply:
It'd be good to track down where it's coming from, ensure that the string is getting extracted there and remove the filter here.

It's possible to do `var notExtractedHere = 'test str'` and `<div>{{notExtractedHere | translate}}</div>` which would do the translation only if the string is extracted in another place which we shouldn't be relying on.